stacks-node/src/tests/signer/v0.rs

```diff
@@ -809,8 +785,11 @@ impl MultipleMinerTest {
             .unwrap()
             .block_height;
 
+        debug!("HERE Building {nmb_blocks} BTC blocks.");
```

Comment:
Probably either remove these lines or replace the "HERE" with something more descriptive, like `boot_to_epoch_3`?